fe/src/pages/Home/Home.tsx

```diff
@@ -27,9 +24,11 @@ const PRODUCTS_CARD_ITEMS: Array<ProductCardItem> = [
     product: {
       id: 0,
       name: 'HP Pavilion 300H',
-      description: 'This is this product description..\n\nStats for HP Pavilion 300H:\nCamera - Good\nQuality - Very bad\n',
+      description:
+        'This is this product description..\n\nStats for HP Pavilion 300H:\nCamera - Good\nQuality - Very bad\n',
```

Comment:
I believe  these and some other lines were not necessary to change.
You can keep them for now, but for future reference it's better to minimize the amount of changed lines. Keeps the diff cleaner and makes it easier to review.